diff --git a/node_modules/typescript/lib/lib.esnext.disposable.d.ts b/node_modules/typescript/lib/lib.esnext.disposable.d.ts
index b169d96..2bc15f0 100644
--- a/node_modules/typescript/lib/lib.esnext.disposable.d.ts
+++ b/node_modules/typescript/lib/lib.esnext.disposable.d.ts
@@ -191,3 +191,6 @@ interface IteratorObject<T, TReturn, TNext> extends Disposable {
 
 interface AsyncIteratorObject<T, TReturn, TNext> extends AsyncDisposable {
 }
+
+interface AsyncIterator<T, TReturn, TNext> extends AsyncDisposable {
+}
